entrypoint.sh

```diff
@@ -1,26 +1,36 @@
 #!/bin/bash
 
+# config
+default_semvar_bump=${DEFAULT_BUMP:-minor}
+
 # get latest tag
-t=$(git describe --tags `git rev-list --tags --max-count=1`)
+tag=$(git describe --tags `git rev-list --tags --max-count=1`)
+tag_commit=commit=$(git rev-list -n 1 $tagag)
```

Comment:
typo? `$tagag` => `$tag`

Reply:
Is this a copy paste error `tag_commit=commit=`? Or is that intentional?

Reply:
Thanks yeah I absolutely messed up before commit 😂 - thanks